test/integration/create_application_project_test.rb

```diff
@@ -0,0 +1,728 @@
+    fill_in 'project_scrn_id', with: 'Testing scrn id'
+    fill_in 'project_programme_approval_date', with: '01/06/2019'
+    fill_in 'project_phe_contacts', with: 'Testing phe contacts'
+    fill_in 'project_acg_who', with: 'Testing acg who'
+    select 'S251 Regulation 2', from: 'project_s251_exemption_id'
+    fill_in 'project_cag_ref', with: 'Testing cag ref'
+    fill_in 'project_date_of_renewal', with: '01/05/2019'
+    check 'Art. 6.1(a)'
+    check 'Art. 6.1(b)'
+    check 'Art. 6.1(c)'
+    check 'Art. 6.1(d)'
+    check 'Art. 6.1(e)'
+    check 'Art. 6.1(f)'
+    check 'Art. 9.2(a)'
+    check 'Art. 9.2(b)'
+    check 'Art. 9.2(c)'
+    check 'Art. 9.2(d)'
+    check 'Art. 9.2(e)'
+    check 'Art. 9.2(f)'
+    check 'Art. 9.2(h)'
+    check 'Art. 9.2(i)'
+    check 'Art. 9.2(j)'
+    fill_in 'project_ethics_approval_nrec_name', with: 'Testing ethics approval nrec name'
+    fill_in 'project_ethics_approval_nrec_ref', with: 'Testing ethics approval nrec ref'
+    select 'UK', from: 'project_processing_territory_id'
+    fill_in 'project_processing_territory_other', with: 'Testing processing territory other'
+    fill_in 'project_dpa_org_code', with: 'Testing dpa org code'
+    fill_in 'project_dpa_org_name', with: 'Testing dpa org name'
+    fill_in 'project_dpa_registration_end_date', with: '01/06/2022'
+    select 'ISO 27001', from: 'project_security_assurance_id'
+    fill_in 'project_ig_code', with: 'Testing ig code'
+    fill_in 'project_data_processor_name', with: 'Testing data processor name'
+    fill_in 'project_data_processor_add1', with: 'Testing data processor add1'
+    fill_in 'project_data_processor_add2', with: 'Testing data processor add2'
+    fill_in 'project_data_processor_city', with: 'Testing data processor city'
+    fill_in 'project_data_processor_postcode', with: 'Testing data processor postcode'
+    select 'UNITED KINGDOM', from: 'project_data_processor_country_id'
+    select 'UK', from: 'project_processing_territory_outsourced_id'
+    fill_in 'project_processing_territory_outsourced_other',
+            with: 'Testing processing territory outsourced other'
+    fill_in 'project_dpa_org_code_outsourced', with: 'Testing dpa org code outsourced'
+    fill_in 'project_dpa_org_name_outsourced', with: 'Testing dpa org name outsourced'
+    fill_in 'project_dpa_registration_end_date_outsourced', with: '01/04/2019'
+    select 'ISO 27001', from: 'project_security_assurance_outsourced_id'
+    fill_in 'project_ig_code_outsourced', with: 'Testing ig code outsourced'
+    fill_in 'project_ig_toolkit_version_outsourced', with: 'Testing ig toolkit version outsourced'
+    fill_in 'project_additional_info', with: 'Testing additional info'
+    click_button 'Create Application'
+    assert has_content?('Application was successfully created.')
+    click_button 'Submit'
+
+    project = Project.find_by(name: 'New application test')
+
+    assert project.owner == User.find_by(username: 'standarduser1')
```

Comment:
Can we please change these all from:

```ruby
assert actual == expected
```

to

```ruby
assert_equal expected, actual
```

to be more idiomatic, and improve the error output please? It'll allow the test runner to say things like "expected 'a', got 'b', rather than "expected false to be true".